data/schema.yaml

```diff
@@ -365,14 +400,35 @@ components:
             title:
               type: string
               example: "Dramatic capsize in America's Cup yacht"
-            versionId:
+            type:
+              enum:
+                - aresMediaMetadata
               type: string
-              example: 'p01jvmg7'
-            versionType:
-              type: string # probably should be an enum in the future
-              example: 'Original'
-            warnings:
-              type: object
+            versions:
+              type: array
+              items:
```

Comment:
Are there any required values for each item in the versions array?

Reply:
added on 411